mitielib/include/mitie.h

```diff
@@ -1041,8 +1041,10 @@ extern "C"
         requires
             - twfe != NULL
             - word == a NULL terminated C string.
+            - result is an empty array of floats. The pointer to the array should be passed 
```

Comment:
What is an array?  C doesn't really have arrays.  The syntax `float thing[]` is just another way of writing `float* thing`.  So this should say something like result points to a block of memory.  How much memory?